apps/heureka/src/components/Services/ServicesList/ServicesList.test.tsx

```diff
@@ -4,24 +4,56 @@
  */
 
 import React from "react"
-import { render, screen, act } from "@testing-library/react"
+import { render, screen, act, fireEvent } from "@testing-library/react"
```

Comment:
`fireEvent` is [deprecated](https://testing-library.com/docs/user-event/intro#differences-from-fireevent) we should use `userEvent` instead. You can see its usage in [Shell.test.tsx](https://github.com/cloudoperators/juno/blob/495b6074712d7e4e6471949af255a86482dcfa91/apps/heureka/src/components/Shell/Shell.test.tsx#L8)